### PR TITLE
Guide on the lightweight runtime works with kubeedge

### PR DESCRIPTION
**Pull request by chendave**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by chendave** [comment]
cc @kevin-wangzefeng  @fisherxu  @hanneshoettinger

File: docs/setup/kubeedge_cri_configure.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# KubeEdge lightweight runtime configuration
+
+## ContainerD
```
Comment (kevin-wangzefeng) [inline docs/setup/kubeedge_cri_configure.md:3]:
```suggestion
## containerd
```

File: docs/setup/kubeedge_configure.md [hunk 2]
```diff
@@ -202,15 +202,7 @@ Verify the configurations before running `edgecore`
...
-5. If your runtime-type is remote, specify the following parameters for remote/CRI based runtimes
-
-    ```yaml
-    remoteRuntimeEndpoint: /var/run/containerd/containerd.sock
-    remoteImageEndpoint: /var/run/containerd/containerd.sock
-    runtimeRequestTimeout: 2
-    podSandboxImage: k8s.gcr.io/pause
-    kubelet-root-dir: /var/run/kubelet/
-    ```
+5. If your runtime-type is remote, follow this guide[KubeEdge CRI Configuration](kubeedge_cri_configure.md) to setup KubeEdge with the remote/CRI based runtimes
```
Comment (kevin-wangzefeng) [inline docs/setup/kubeedge_configure.md:205]:
```suggestion
5. If your runtime-type is remote, follow this guide [KubeEdge CRI Configuration](kubeedge_cri_configure.md) to setup KubeEdge with the remote/CRI based runtimes
```

File: docs/setup/kubeedge_cri_configure.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+```
+
+*NOTE: namespace here might be changed soon from “k8s.io” to “kubeedge.io”.*
+
+
+
+
+## CRI-O
+
+Follow the [CRI-O install guide] (https://github.com/cri-o/cri-o/blob/master/tutorials/setup.md) to setup CRI-O.
```
Comment (kevin-wangzefeng) [inline docs/setup/kubeedge_cri_configure.md:75]:
```suggestion
Follow the [CRI-O install guide](https://github.com/cri-o/cri-o/blob/master/tutorials/setup.md) to setup CRI-O.
```

File: docs/setup/kubeedge_cri_configure.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+```bash
+git clone https://github.com/cri-o/cri-o
+cd cri-o
+make
+sudo make install
+# generate and install configuraion files
+sudo make install.config
+```
+
+Setup CNI networking, please follow this guide [setup CNI] (https://github.com/cri-o/cri-o/blob/master/contrib/cni/README.md) to set up the CNI networking.
```
Comment (kevin-wangzefeng) [inline docs/setup/kubeedge_cri_configure.md:88]:
```suggestion
Setup CNI networking, please follow this guide [setup CNI](https://github.com/cri-o/cri-o/blob/master/contrib/cni/README.md) to set up the CNI networking.
```

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve

Thanks

**Pull request merged**